### PR TITLE
Some minor update in readme

### PR DESCRIPTION
**Pull request by annafariha**
This PR has some update in readme.

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -5,21 +5,19 @@ This project consists of two components:
...
 
-Both of these components are found in the `ai-compile` directory.
+Both of these components are in the `ai-compile` directory.
 
 ## Pre-requisities
 
 - Python 3.8.9 (64-bit). Python 3.9 does not work due to tensorflow requirement.
 - Visual Studio 2019 with Desktop development with C++.
-- Some recent version of node and npm for building the vscode-extension.
+- Recent version of node and npm for building the vscode-extension.
```
Comment (annafariha) [inline BUILD.md:14]: Can we specify exact versions that are compatible? "Some recent version" is vague and confusing.

File: BUILD.md [hunk 2]
```diff
@@ -53,17 +51,18 @@ The response should have some useful data.
```
Comment (annafariha) [inline BUILD.md:51]: I don't know what is meant by this. We need to be either specific or just get rid of the RESP server testing part. I am not even sure how to exactly do this. If we want to keep it, let's give guidelines on how to do it.

File: BUILD.md (hunk 1 above)
Comment (barik) [reply BUILD.md:14]: I have no idea which exact versions are compatible.

Comment (annafariha) [reply BUILD.md:14]: That's what I assumed and hence just removed "Some".

**Pull request merged**